apps/lend/src/components/ChartOhlcWrapper/LiquidityData.tsx

```diff
@@ -0,0 +1,191 @@
+import type { LlammaControllerEvent } from '@/ui/Chart/types'
+import { LendingMarketTokens } from './types'
+
+import styled from 'styled-components'
+import { t } from '@lingui/macro'
+
+import networks from '@/networks'
+import { formatNumber, getFractionDigitsOptions } from '@/ui/utils'
+import { convertFullTime, convertTime, convertTimeAgo } from '@/components/ChartOhlcWrapper/utils'
+
+import Box from '@/ui/Box'
+import TokenIcon from '@/components/TokenIcon'
+import { Chip } from '@/ui/Typography'
+import Tooltip from '@/ui/Tooltip'
+
+type Props = {
+  lendControllerData: LlammaControllerEvent[]
+  chainId: ChainId
+  coins: LendingMarketTokens
+}
+
+const LiquidityData = ({ lendControllerData, chainId, coins }: Props) => {
```

Comment:
Use React's native typing so that TS immediately understands that it's a component
```suggestion
const LiquidityData: React.FC<LiquidityDataProps> = ({ lendControllerData, chainId, coins }) => {
```